web/src/settings_users.js

```diff
@@ -249,6 +253,10 @@ function bot_info(bot_user_id) {
     // It's always safe to show the real email addresses for bot users
     info.display_email = bot_user.email;
 
+    if (owner_id) {
+        info.owner_img_src = people.small_avatar_url_for_person(people.get_by_user_id(owner_id));
+    }
```

Comment:
Can we merge this condition check here?
https://github.com/zulip/zulip/blob/63398eaeb9a22425b1e63869f470f5e4d97cc227/web/src/settings_users.js#L229-L231